cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,605 @@
+                vm,
+                self.migration_request_details_list.is_in_progress(vm),
+            )
+            migration_plan_in_progress_tracker.append(
+                self.migration_request_details_list.is_in_progress(vm)
+                and (clock_reading1 < clock_reading2)
+            )
+        return not any(migration_plan_in_progress_tracker)
+
+
+@attr.s
+class MigrationPlan(BaseEntity):
+    """Class representing v2v Migration Plan"""
+
+    name = attr.ib()
+
+    def is_plan_started(self):
+        view = navigate_to(self, "InProgress")
+        return wait_for(
+            func=view.progress_card.is_plan_started,
+            func_args=self.name,
+            message="migration plan is starting, be patient please",
+            delay=5,
+            num_sec=150,
+            handle_exception=True,
+            fail_cond=False,
+        )
+
+    def is_plan_in_progress(self):
+        """MIQ V2V UI is going through redesign as OSP will be integrated."""
+        view = navigate_to(self, "InProgress")
+
+        def _in_progress():
+            try:
+                is_plan_visible = view.progress_card.is_plan_visible(self.name)
+                if is_plan_visible:
+                    try:
+                        plan_time_elapsed = view.progress_card.get_clock(self.name)
+                        new_msg = "time elapsed for migration: {time}".format(
+                            time=plan_time_elapsed
+                        )
+                    except NoSuchElementException:
+                        new_msg = "playbook is executing.."
+                        pass
+                    logger.info(
+                        "For plan {plan_name}, is plan in progress: {visibility},"                                
+                        " {message}".format(
+                            plan_name=self.name, visibility=is_plan_visible, message=new_msg
+                        )
+                    )
+                return not is_plan_visible
+            except ItemNotFound:
+                return True
+            
+    def is_migration_complete(self, name):
+        """Uses search box to find migration plan, return True if found.
+        Args:
+            migration_plan name: (object) Migration Plan name
+        """
+        view = navigate_to(self, "Complete")
+        view.wait_displayed()
+        if self.appliance.version >= "5.10":
+            view.items_on_page.item_select("15")
+            while not view.clear_filters.is_displayed:
+                view.switch_to("Completed Plans")
+                view.search_box.fill("{}\n\n".format(name))
+        return name in view.plans_completed_list.read() and view.plans_completed_list.is_plan_succeeded(
+            name
+        )
+
+    def migration_plan_request(self, name):
+        view = navigate_to(self, "InProgress")
+        view.progress_card.select_plan(name)
+        request_view = self.create_view(MigrationPlanRequestDetailsView)
```

Comment:
Required:
use 
`request_view = self.create_view(MigrationPlanRequestDetailsView, wait="10s")`
and remove `request_view.wait_displayed()`